README.md

```diff
@@ -225,6 +225,56 @@ shareBase64({
 });
 ```
 
+### downloadBase64
+
+<kbd>App version >=24.10</kbd> <kbd>App version >=24.9 in O2ES</kbd>
+
+Opens the provided file using the download webview mode. The file is provided as
+a base64 encoded string.
+
+```ts
+downloadBase64: ({contentInBase64: string; fileName: string}) => Promise<void>;
+```
+
+-   The file type will be inferred from the `fileName` extension. The file
+    extension is mandatory. Take into account that iOS webview won't be able to
+    render file types not supported by Safari.
+
+#### Behaviour
+
+##### Android
+
+1. Once file is correctly processed, a "Downloaded" notification is shown in the
+   system notifications inbox. System will try to open the file when clicking on
+   it.
+2. Simultaneously, app will try to open the given file, this may result in 3
+   situations:
+    - No app that can handle this type of file is available
+        - Nothing will happen, user feedback will be just the previous generated
+          notification.
+    - Multiple apps can handle this type of file
+        - System will show a desambiguator window to select the app which will
+          be used to open the file.
+    - Single app can open this type of file (Or an app is set as default for
+      these kind of files)
+        - App will be presented immediatly with the donwloaded content.
```

Comment:
```suggestion
        - Downloaded content will be opened using the only available App that supports its extension.
```

Reply:
The suggestion is a rephrasing that feels more natural to me. If you still prefer the current sentence, please fix `immediatly` and `donowloaded` typos :)